### DISCUSSION TITLE
Needs example for simulcast

### DISCUSSION THREAD

**Discussion by jinleileiking** [pion/webrtc#2494] [Q&A]
I found this example:

https://github.com/pion/webrtc/tree/master/examples/simulcast

But this does not for poin generating simulcasting.

from test:

https://github.com/pion/webrtc/blob/2ffab965d3da4e5ab92473964b2de669a3a1518a/peerconnection_media_test.go#L1216

got some clue. but using this code is too bad. 

Need more elegant codes.

**Comment by jinleileiking** [comment]
Need api like:

https://github.com/livekit/server-sdk-go/blob/34cd02a152ec8a695274a4c089348f803fbc73ed/localsampletrack.go#LL265C21-L265C21

**Reply by func2** [reply]
@jinleileiking Sorry can you provide a simple example how to dynamically choose simulcast video layer which pion libraries?
